.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,10 +9,12 @@ Make sure these boxes are checked before your pull request (PR) is ready to be r
 * [ ] screenshots/GIFs are attached 📎 in case of UI updation
 * [ ] ask `@publiclab/reviewers` for help, in a comment below
 
-> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!
+<!--We're happy to help you get this ready -- don't be afraid to ask for help, and don't be discouraged if your tests fail at first!-->
 
-If tests do fail, click on the red `X` to learn why by reading the logs.
 
-Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 
+<!--If tests do fail, click on the red X to learn why by reading the logs.-->
 
-Thanks!
+<!-- Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
```

Comment:
I think you need to add a closing bracket here, at the end?

Reply:
@anirudhprabhakaran3 Wow, great catch! Thanks for the code review.

@DalilaBA please go ahead and make the change if you can!